### DISCUSSION TITLE
Request to transfer ownership of nix4vscode to nix-community

### DISCUSSION THREAD

**Discussion by cathaysia** [nix-community/infra#835] [Show and tell]
As a result of talke https://github.com/nix-community/nix-vscode-extensions/issues/47 ,it's seem it would be a good idea to transfer ownership of [nix4vscode](https://github.com/cathaysia/nix4vscode) to nix-community. But I don't have permission now. If you agree with this decision. Please give me relevant permissions

**Comment by zowoq** [comment]
I've invited you to the nix-community org and created a team @nix-community/nix4vscode for this project.

**Reply by cathaysia** [reply]
thanks. :)

**Comment by cathaysia** [comment]
@spikespaz @deemp Are you interested in joining this team?

**Reply by spikespaz** [reply]
I believe I would be, yes.

**Reply by cathaysia** [reply]
@zowoq Can you invite him to join the organization?

**Reply by zowoq** [reply]
Invite sent.

**Reply by deemp** [reply]
Yes, I'm interested.
